<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>busqueda</class>
 <widget class="QDialog" name="busqueda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar Registro Por:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_busqueda">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lb_busqueda">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_busqueda">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="lb_info">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Su busqueda ha sido realizada, favor cerrar esta ventana para observar resultados!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
